core/src/OpenACC/Kokkos_OpenACC_Team.hpp

```diff
@@ -257,26 +256,29 @@ class TeamPolicyInternal<Kokkos::Experimental::OpenACC, Properties...>
         m_thread_scratch_size(p.m_thread_scratch_size),
         m_tune_team_size(p.m_tune_team_size),
         m_tune_vector_length(p.m_tune_vector_length),
-        m_chunk_size(p.m_chunk_size) {}
+        m_chunk_size(p.m_chunk_size),
+        m_space(p.m_space) {}
 
   /** \brief  Specify league size, request team size */
-  TeamPolicyInternal(const typename traits::execution_space&,
+  TeamPolicyInternal(const typename traits::execution_space& space_,
```

Comment:
While the trailing underscore?

Reply:
I just followed the pattern in the CUDA backend implementation; should it be deleted?

Reply:
Removed.